test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj

```diff
@@ -425,6 +427,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		F4A4D85E28A599050015ED63 /* XCRemoteSwiftPackageReference "mobile-native-ios" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/iTwin/mobile-native-ios";
+			requirement = {
+				kind = exactVersion;
+				version = 3.2.14;
```

Comment:
Will this need to be updated each time we publish a new addon? If so please update the two [wiki articles](https://dev.azure.com/bentleycs/iModelTechnologies/_wiki/wikis/iModelTechnologies.wiki/2036/Creating-a-new-imodel.js-addon).

Reply:
thx, updated